pkg/autoscaler/multiscaler.go

```diff
@@ -48,7 +49,10 @@ type Metric struct {
 
 // MetricSpec is the parameters in which the Revision should scaled.
 type MetricSpec struct {
-	TargetConcurrency float64
+	TargetConcurrency      float64
+	TargetConcurrencyPanic float64
+	Window                 time.Duration
```

Comment:
Should this be qualified as `StableWindow` vs. `PanicWindow` (or `WindowStable` if that's nicer)

Reply:
How about these for the 4 dimensions? 

```
StableTargetConcurrency
PanicConcurrencyThresholdPercentage
StableWindow
PanicWindowPercentage
```

Reply:
Yes, these 4 make more sense to me 👍 . Thanks!

Reply:
@markusthoemmes Renamed all the variables as specified above.  Note that in the Metric the panic dimensions are materialized (not a percentage).  I also used a shorter form in the names for the annotations.  But in general they should make more sense to you now.